recipes/phytop/src/small_tools.py

```
@@ -0,0 +1,358 @@
+		for value in temp:
+			# try: value = value.strip().decode('gbk')
+			# except UnicodeDecodeError: value = 'NULL'
+			ws1.write(i,j,value)
+			j += 1
+		i += 1
+	wb.save(xls_file)
+
+def combine_tabs_2xls(table_files, xls_file = None, sheets = None):
+	# init
+	if not xls_file:
+		xls_file = table_file[0] + '.xls'
```

Ruff

Undefined name `table_file`

(F821)